Assets/LeapMotionModules/GraphicRenderer/Scripts/LeapGraphicGroup.cs

```diff
@@ -479,8 +557,14 @@ public partial class LeapGraphicGroup : ISerializationCallbackReceiver {
     public void OnBeforeSerialize() { }
 
     public void OnAfterDeserialize() {
-      _renderingMethod.Value.group = this;
-      _renderingMethod.Value.renderer = renderer;
+      if (_renderingMethod.Value == null || renderer == null) {
+        Debug.LogWarning("The rendering group did not find the needed data!  If you have a variable of type " +
+                         "LeapGraphicGroup make sure to annotate it with a [NonSerialized] attribute, or else " +
+                         "Unity will automatically create invalid instances of the class.");
```

Comment:
oh my! Would this result in anything being in a bad or difficult-to-recover state? mostly just curious